### DISCUSSION TITLE
How can I delete an icon from the gallery?

### DISCUSSION THREAD

**Discussion by ansgar75** [pixelit-project/PixelIt#372] [Q&A]
Hello,

Unfortunately I created an icon with wrong pixels. How can I delete this icon or is there an option to adapt an existing icon (made by myself)?
Thank you.

Kind regards
Ansgar

**Comment by foorschtbar** [comment]
@o0shojo0o could help you :)

**Comment by o0shojo0o** [comment]
You can't delete icons yourself, but I can do it for you. 
Which ID are you talking about?

**Reply by ansgar75** [reply]
Hi. I'm talking about ID 1568.
Thank you.

**Reply by o0shojo0o** [reply]
Done :-)
